server/src/test/java/org/jetbrains/bsp/bazel/server/bloop/BloopExporterTest.kt

```diff
@@ -0,0 +1,33 @@
+package org.jetbrains.bsp.bazel.server.bloop
+
+import ch.epfl.scala.bsp4j.BuildTargetIdentifier
+import org.jetbrains.bsp.bazel.server.bloop.BloopExporter.BazelExportFailedException
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.fail
+
+class BloopExporterTest {
+    @Test
+    fun export_project_with_failed_direct_target_succeeds() {
+        val projectTargets = setOf(BuildTargetIdentifier("a"), BuildTargetIdentifier("b"))
+        val failedTargets = setOf(BuildTargetIdentifier("a"))
+
+        try {
+            BloopExporter.validateNoFailedExternalTargets(projectTargets, failedTargets)
```

Comment:
I wanted to use `assertThrows` etc here but using them caused the kotlin compiler to crash...